README.md

```diff
@@ -81,6 +82,7 @@
 - [Capture the Ether](https://capturetheether.com/) - Game in which you hack Ethereum smart contracts to learn about security.
 - [Ethereum Smart Contract Security Best Practices](https://consensys.github.io/smart-contract-best-practices/) - General security philosophy, known attacks, and sample code.
 - [Not So Smart Contracts](https://github.com/crytic/not-so-smart-contracts) - Examples of common vulnerabilities, including code from real smart contracts.
+- [Best practices for smart contract development](https://yos.io/2019/11/10/smart-contract-development-best-practices/)- Best practices for smart contract development
```

Comment:
This entry already exists [here](https://github.com/bkrem/awesome-solidity/pull/69/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R65), originally added by the author himself.
```suggestion

```